tests/test_lookml.py

```diff
@@ -705,6 +706,24 @@ def msg_glean_probes():
     ]
 
 
+@contextlib.contextmanager
+def pool_started():
+    """
+    Needed when mocking objects that are used within a multiprocessing pool.
+
+    The default start method when using multiprocessing is spawn on MacOS and Windows.
+    If using fork, your processes are forked in the current state, including the mocking,
+    while with using spawn, a new Python interpreter is launched, where the mock will not work.
+    The start method needs to be changed to fork for the mocks to work across processes.
+    """
+    start_method = multiprocessing.get_start_method()
+    try:
+        multiprocessing.set_start_method("fork", force=True)
```

Comment:
The [Python docs](https://docs.python.org/3.10/library/multiprocessing.html#contexts-and-start-methods) say that the `fork` method isn't available for Windows and should be considered unsafe on macOS.  Could we avoid the need to do this somehow? (e.g. mocking `Pool`, or special-casing `parallelism=1` to not use `multiprocessing`)

Reply:
I changed the logic to use a for loop when `parallelism == 1` (which is even slightly faster)